superset-frontend/src/cccs-viz/plugins/plugin-chart-application-links/src/ApplicationLinks.tsx

```diff
@@ -1,53 +1,104 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { ApplicationsProps } from './types';
 import styles from './styles';
 
 export default function ApplicationLinks(props: ApplicationsProps) {
   const { application, appVal, appType } = props;
-  const [alfredCount] = useState(-1);
+  const [alfredCount, setAlfredCount] = useState(-1);
+  const [alfredURL, setAlfredURL] = useState(
+    'https://alfred-pb.chimera.cyber.gc.ca',
+  );
 
-  let url = '';
   let infoType = '';
-  // TODO: this can be re added CLDN-929
-  // let callback_url = '';
+  let callback_url = '';
 
   if (application === 'ALFRED') {
     if (appType === 'USER_ID') {
-      url = `https://alfred-tst.u.chimera.azure.cyber.gc.ca/?expression=MATCH%20(email:EMAIL_ADDRESS)%20WHERE%20email.value%20in%20[%22${appVal}%22]%20return%20email.value,%20email.maliciousness,%20email.uri`;
       infoType = 'user id';
-    } else {
-      url = `https://alfred-tst.u.chimera.azure.cyber.gc.ca/?expression=MATCH%20(ip%3AIP_ADDRESS)%20WHERE%20ip.value%20IN%20%5B%22${appVal}%22%5D%20RETURN%20ip.value%2C%20ip.maliciousness%2C%20ip.creation_date%2C%20ip.created_by%2C%20ip.uri%2C%20ip.report_uri`;
+      callback_url = 'user_id';
```

Comment:
I'm wondering if we should rename this variable. Doesn't look like a URL to me. :)

Reply:
Renamed `callback_url` to `callback_identifier` in latest commit